app/src/main/java/com/example/guineapics/RecyclerViewAdapter.java

```diff
@@ -0,0 +1,43 @@
+package com.example.guineapics;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.example.guineapics.dao.GuineaPigsDAO;
+
+public class RecyclerViewAdapter extends RecyclerView.Adapter<TileViewHolder> {
+    private LayoutInflater inflater;
+    private GuineaPigsDAO pigsDAO = GuineaPigsDAO.getDAO();
+
+    //TODO: should I store the list or is DAO acceptable?
+    public RecyclerViewAdapter(Context context /*, ArrayList<GuineaPig> guineaPigs*/) {
+        this.inflater = LayoutInflater.from(context);
```

Comment:
no need to pass context, you have it already in views

Reply:
what do you mean?